src/base/bittorrent/trackerentrystatus.h

```diff
@@ -59,8 +60,8 @@ namespace BitTorrent
         int numLeeches = -1;
         int numDownloaded = -1;
 
-        QDateTime nextAnnounceTime {};
-        QDateTime minAnnounceTime {};
+        AnnounceTimePoint nextAnnounceTime {};
+        AnnounceTimePoint minAnnounceTime {};
```

Comment:
> They can init to zero by default

... just like many other fields of this structure. However, we add initializers to avoid compiler warnings when we use a reduced initializer list to create `TrackerEntryStatus`.